dataretrieval/waterdata/api.py

```diff
@@ -1,107 +1,62 @@
-        "narrow",
-        "resultdetectionquantitationlimit",
-        "labsampleprep",
-        "count",
-    ],
-}
+from dataretrieval.waterdata.types import (
+    CODE_SERVICES,
+    PROFILE_LOOKUP,
+    PROFILES,
+    SERVICES,
+)
+from dataretrieval.waterdata.utils import SAMPLES_URL, get_ogc_data
+
+# Set up logger for this module
+logger = logging.getLogger(__name__)
+
 
 def get_daily(
-        monitoring_location_id: Optional[Union[str, List[str]]] = None,
-        parameter_code: Optional[Union[str, List[str]]] = None,
-        statistic_id: Optional[Union[str, List[str]]] = None,
-        properties: Optional[List[str]] = None,
-        time_series_id: Optional[Union[str, List[str]]] = None,
-        daily_id: Optional[Union[str, List[str]]] = None,
-        approval_status: Optional[Union[str, List[str]]] = None,
-        unit_of_measure: Optional[Union[str, List[str]]] = None,
-        qualifier: Optional[Union[str, List[str]]] = None,
-        value: Optional[Union[str, List[str]]] = None,
-        last_modified: Optional[str] = None,
-        skipGeometry: Optional[bool] = None,
-        time: Optional[Union[str, List[str]]] = None,
-        bbox: Optional[List[float]] = None,
-        limit: Optional[int] = None,
-        max_results: Optional[int] = None,
-        convertType: bool = True
-    ) -> pd.DataFrame:
-    """Daily data provide one data value to represent water conditions for the day.
-    Throughout much of the history of the USGS, the primary water data available was
-    daily data collected manually at the monitoring location once each day. With
-    improved availability of computer storage and automated transmission of data, the
-    daily data published today are generally a statistical summary or metric of the
-    continuous data collected each day, such as the daily mean, minimum, or maximum
-    value. Daily data are automatically calculated from the continuous data of the same
-    parameter code and are described by parameter code and a statistic code. These data
-    have also been referred to as “daily values” or “DV”.
+    monitoring_location_id: Optional[Union[str, List[str]]] = None,
+    parameter_code: Optional[Union[str, List[str]]] = None,
+    statistic_id: Optional[Union[str, List[str]]] = None,
+    properties: Optional[List[str]] = None,
+    time_series_id: Optional[Union[str, List[str]]] = None,
+    daily_id: Optional[Union[str, List[str]]] = None,
+    approval_status: Optional[Union[str, List[str]]] = None,
+    unit_of_measure: Optional[Union[str, List[str]]] = None,
+    qualifier: Optional[Union[str, List[str]]] = None,
+    value: Optional[Union[str, List[str]]] = None,
+    last_modified: Optional[str] = None,
+    skip_geometry: Optional[bool] = None,
+    time: Optional[Union[str, List[str]]] = None,
+    bbox: Optional[List[float]] = None,
+    limit: Optional[int] = None,
+    max_results: Optional[int] = None,
+    convert_type: bool = True,
+) -> pd.DataFrame:
```

Comment:
```suggestion
) -> Tuple[pd.DataFrame, BaseMetadata]:
```
Is there a reason the metadata function is a Tuple and this one isn't?